src/lib/y2storage/proposal/base.rb

```diff
@@ -77,6 +78,7 @@ def propose
         result = calculate_proposal
         return result if devices.nil? || devices.empty?
         log.info("Proposed devicegraph:\n\n#{devices.to_str}\n")
+        DumpManager.dump(devices, "proposed")
```

Comment:
Is this required? In dialogs/proposal, the "proposal" and its actions are already saved. This "proposed" is the same devicegraph, isn't it?

Reply:
We can rely on wishful thinking, but I prefer to be on the safe side; see also my long comment above.